src/intTest/java/uk/nhs/digital/nhsconnect/lab/results/inbound/InboundMeshQueueMultiMessageTest.java

```diff
@@ -77,7 +82,17 @@ private void assertGpOutboundQueueMessages(SoftAssertions softly, Message messag
 
         final String messageBody = parseTextMessage(message);
         final String expectedMessageBody = new String(Files.readAllBytes(getFhirResource().getFile().toPath()));
-        softly.assertThat(messageBody).isEqualTo(expectedMessageBody);
+
+        JSONAssert.assertEquals(
+            expectedMessageBody,
+            messageBody,
+            new CustomComparator(
+                JSONCompareMode.STRICT,
+                new Customization("meta.lastUpdated", (c1, c2) -> true),
+                new Customization("identifier.value", (c1, c2) -> true),
+                new Customization("entry[*].fullUrl", (c1, c2) -> true)
```

Comment:
This means it will ignore differences in those nodes, right?

Reply:
Yep. We can't really assert dates or randomly generated values (we can at a unit level), but we can test everything else at integration level

Reply:
In that case, a utility `BiPredicate<?, ?> IGNORE` could make this read better.